contrib/auto_explain/expected/auto_explain_optimizer.out

```diff
@@ -78,7 +76,7 @@ LOG:  statement: SET auto_explain.log_verbose = TRUE;
 -- this select should not dump execution plan
 SELECT relname FROM pg_class WHERE relname='pg_class';
 LOG:  statement: SELECT relname FROM pg_class WHERE relname='pg_class';
-"Falling back to Postgres-based planner because GPORCA does not support the following feature: Queries on coordinator-only tables",
```

Comment:
Hmm...isn't `coordinator-only` better ? I remember that in all code and documents in gp `master` has been changed to `coordinator`.

Reply:
> [greenplum-db/gpdb-archive@a19c9e9448](https://github.com/greenplum-db/gpdb-archive/commit/a19c9e9448) current cbdb have not cherry-pick this PR which will cause a lots of conflict in CBDB.

Got it!